src/FileSystem/DefaultFileSystem.h

```diff
@@ -9,6 +9,7 @@
 
 #pragma once
 
+#include <istream>
```

Comment:
```suggestion
```
Same. Since we use `std::unique_ptr<std::istream>` in the header, it can be forward declared.